strr-web/tests/unit/pages/rental-application.spec.ts

```diff
@@ -36,16 +37,19 @@ describe('Rental Application', () => {
     const stepper = wrapper.findComponent(Stepper)
     expect(stepper.exists()).toBeTruthy()
 
+    expect(wrapper.findComponent(BcrosFormSectionPropertyManagerForm).exists()).toBeTruthy()
+
+    await goToStep(2)
     expect(wrapper.findComponent(BcrosFormSectionContactInformationForm).exists()).toBeTruthy()
     expect(wrapper.findComponent(H2).text()).toContain('Step 1')
```

Comment:
If this is Step 2, then it should contain Step 2 label and not Step 1?

Reply:
Fixed